app/assets/javascripts/poller.js

```diff
@@ -33,13 +33,17 @@
 
       if (prop) {
         $(document).on(
-          prop.replace(/[H|h]idden/, '') + 'visibilitychange',
+          this.visibilityChangeEvent(prop),
           this.handleVisibilityChange.bind(this)
         );
       }
     },
 
-    isHidden: function() {
+    visibilityChangeEvent: function(prop) {
+      return prop.replace(/[H|h]idden/, '') + 'visibilitychange';
```

Comment:
JS supports flags for regex so you could express this case-insensitively as `/hidden/i`.